src/Hooks/useForm.js

```diff
@@ -0,0 +1,8 @@
+import { useState, useCallback } from 'react'
+
+export const useForm = (initValue = '') => {
```

Comment:
This custom hook is simple but effective. Maybe you could try to handle more Form data in here, for example have a values array and handling errors and loading. Just a suggestion to really get the most out of React.